vidhill/the-starry-night
# Use golangci to run linters

- Add config file for golangci-lint
- Run golangci-lint from lint make target
- Add check to see if golangci-lint is installed
- Testing how disabling rules works
- Enable nolintlint to stop people from disabling rules carelessly
- Enable nolintlint to stop people from disabling rules carelessly
- remove test which does nothing
- Modify hooks setup make target to use make
- Add new make target for pre-push
- Remove custom gofmt bash script as golangci-lint now runs it
- Remove check for staticcheck and forbidigo as they are included with golang-ci
- Enable staciccheck linter to run from within golang-ci
- Enable staciccheck linter to run from within golang-ci
- Update README to change dependencies, no longer directly depend on linters
